crates/transaction-emitter/src/lib.rs

```diff
@@ -1070,6 +1102,9 @@ impl Distribution<InvalidTransactionType> for Standard {
 
 fn gen_rng_for_reusable_account(count: usize) -> Vec<StdRng> {
     // use same seed for reuse account creation and reuse
+    // TODO: Investigate why we use the same seed and then consider changing
```

Comment:
This is needed so that we can generate the same set of accounts for rerun of the txn emitter. Can you clarify what issue you are running into?

Reply:
There is an issue where if I run the txn emitter multiple times quickly, the second run can fail because there is already stuff in mempool for the given accounts. This wouldn't happen if the accounts were different. Not a hard blocker but a problem we should consider fixing at some point.

Reply:
What command are you running?

Reply:
I'm following the janky guide I wrote up here: https://www.notion.so/aptoslabs/Single-node-testnet-transaction-load-test-guide-57e82a909e614243bed9694641f581f9. For step 3 though, I was using like `--workers-per-ac 4 --accounts-per-client 4 --wait-millis 0` so it would blow the mempool limit per account. On the next run I would see an error either like this: https://gist.github.com/banool/b67ba3b6d3095ee5038a4ada976af2a4 or of another type (but I can't find it / repro right now).

Reply:
@banool - You shouldn't really hit this issue, IIUC, the reusable account path is triggered only when `vasp` flag is true. Are you setting that flag to true?

Reply:
I was not using `vasp`, I think it's just because in these situations mempool already had stuff leftover from the previous run.